apisix/cli/file.lua

```diff
@@ -65,8 +65,15 @@ local function resolve_conf_var(conf)
             local var_used = false
             -- we use '${{var}}' because '$var' and '${var}' are taken
             -- by Nginx
-            local new_val = val:gsub("%$%{%{%s*([%w_]+)%s*%}%}", function(var)
-                local v = getenv(var)
+            local new_val = val:gsub("%$%{%{%s*([%w_]+[%:%w+]*)%s*%}%}", function(var)
```

Comment:
Could you add a test in https://github.com/apache/apisix/blob/b8daf65b3c0690138feccb2ce399fc58349ac7ee/t/cli/test_main.sh#L311?
Also need to update the doc in https://github.com/apache/apisix/blob/11433d740c5e19b757c61f59842d6de64e27fba8/conf/config.yaml#L24

Reply:
OK, I'll try

Reply:
Done